modules/common/react/lib/Popper.tsx

```diff
@@ -15,9 +15,16 @@ export interface PopperProps extends React.HTMLAttributes<HTMLDivElement> {
    */
   children: React.ReactNode;
   /**
-   * The element that contains the portal children when `portal` is true.
+   * The containing element for Popper elements. The Popper uses
+   * {@link https://reactjs.org/docs/portals.html Portals} to place the DOM elements
+   * of the Popper in a different place in the DOM to prevent issues with overflowed containers.
+   * When the popper is opened, `aria-hidden` will be added to siblings to hide background
+   * content from assistive technology like it is visibly hidden from sighted users. This property
+   * should be set to the element that the application root goes - not containing element of content.
+   * This should be a sibling or higher than the header and navigation elements of the application.
+   * @default document.body
    */
-  containerElement?: Element;
```

Comment:
probably keep this as element?

Reply:
`export function createPortal(children: ReactNode, container: Element, key?: null | string): ReactPortal;`
^ Types from React.createPortal, so yeah - let's keep as `Element`